service/assessment/assessment.go

```diff
@@ -166,56 +165,69 @@
 		s.informHooks(nil, newError)
 
 		res = &assessment.AssessEvidenceResponse{
-			Status: false,
+			Status:        assessment.AssessEvidenceResponse_FAILED,
+			StatusMessage: newError.Error(),
 		}
 
-		return res, status.Errorf(codes.InvalidArgument, "invalid req: %v", newError)
+		return res, status.Error(codes.InvalidArgument, "invalid evidence")
```

Comment:
We could do it like that. The error message including the Validate()-error will be returned in the case of calling AssessEvidence() and in case of AssessEvidences() the error is stored in the statusMessage in the response. 

```suggestion
		return res, status.Error(codes.InvalidArgument, newError.Error())
```